frontend/routes/components/resources/attachResourcesDialog/attachResourceFiles.component.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ *  Copyright (C) 2019 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import * as React from 'react';
+import {ResourcesDropzone} from './attachResourcesDropzone';
+import { FieldArray, Field } from 'formik';
+import { TextField } from '@material-ui/core';
+import { RemoveButton } from '../resources.component';
+import { StyledFormControl,
+		FieldsRow } from '../../../viewer/components/risks/components/riskDetails/riskDetails.styles';
```

Comment:
```suggestion
import {
 StyledFormControl,
 FieldsRow
} from '../../../viewer/components/risks/components/riskDetails/riskDetails.styles';
```

Reply:
And tbh, you shouldn't use styles of a different component. The better approach is moving common style to shared space.